app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/check_your_details.scala.html

```diff
@@ -193,7 +193,7 @@
 }
 
 @isUtrAndNinoEmpty = @{
-    utr.forall(_.id == "Not entered") && nino.forall(_.id == "Not entered") && nameIdOrganisationDetails.forall(_.id.isEmpty)
+    utr.forall(_.id == messages("cds.not-entered.label")) && nino.forall(_.id == messages("cds.not-entered.label")) && nameIdOrganisationDetails.forall(_.id.isEmpty)
```

Comment:
Shall we add a test to CheckYourDetailsSpec to make sure it won't break again?

Reply:
I checked the tests and there is no easy way implemented to test switch of the languages. All view tests are based on EN language only.

Reply:
Actually added the test for the case it was failing, was not that difficult to implement.